### ISSUE TITLE
Router sets body children invisible - Using templates and meteor packages (eg. datepickers) difficult.

### ISSUE THREAD

**Issue by aido179** [gterrono/houston#355]
See: 
https://github.com/gterrono/houston/blob/c5994e164d33e6dd719ee0aa97234e868d2c7765/client/router.coffee#L92-L94

The router is hiding "non-admin stuff" - surely it should remove them from the DOM. 

Using packages that create dynamic content within the template, for example datepickers, ends up really hard to debug as they are set with visibility: hidden.

**Comment by gterrono** [comment]
Yeah, that makes a lot of sense. @AlexeyMK do you remember if there was a reason why we didn't do that? Otherwise, it should be a pretty quick fix. I can probably get to it tonight.

**Comment by aido179** [comment]
Excellent. Thanks for quick reply. I'd do it myself but I'm not familiar with coffee script.
